README.md

```diff
@@ -20,6 +20,12 @@ In order for this app to work, you will need to create a [Firebase project](http
 * [Enable Anonymous Authentication](https://firebase.google.com/docs/auth/android/anonymous-auth#before-you-begin) in your Firebase project
 * [Enable Email/Password Authentication](https://firebase.google.com/docs/auth/android/password-auth#before_you_begin) in your Firebase project
 * Run the app using Android Studio Flamingo+ on a device/emulator with API level 21 or above
+* Create your first to-do item, then open the Firestore dashboard on the Firebase console
+* Navigate to the Indexes tab and create a new index with the following configuration:
```

Comment:
I'd avoid context switching in the same step (ie. first do X in the app then do Y in the console)
What if we only mention the app in the first step and then mention the console in a second step?

Also, we can deep link to the console for the indexes tab.

I added both suggestions:

```suggestion
* Create your first to-do item in the app
* In the Firebase console, navigate to the [Firestore Indexes tab](https://console.firebase.google.com/project/_/firestore/indexes) and create a new index with the following configuration:
```

Reply:
This looks great, thanks! I'll commit your suggestions.